[2025-02-03T13:58:45.353+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-02-03T13:58:45.410+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: INETUM_ETL.extract_data manual__2025-02-03T13:58:38.091646+00:00 [queued]>
[2025-02-03T13:58:45.438+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: INETUM_ETL.extract_data manual__2025-02-03T13:58:38.091646+00:00 [queued]>
[2025-02-03T13:58:45.442+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 2
[2025-02-03T13:58:45.497+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): extract_data> on 2025-02-03 13:58:38.091646+00:00
[2025-02-03T13:58:45.519+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=119) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-03T13:58:45.519+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'INETUM_ETL', 'extract_data', 'manual__2025-02-03T13:58:38.091646+00:00', '--job-id', '267', '--raw', '--subdir', 'DAGS_FOLDER/dagto_csv.py', '--cfg-path', '/tmp/tmp52cojzru']
[2025-02-03T13:58:45.524+0000] {standard_task_runner.py:63} INFO - Started process 121 to run task
[2025-02-03T13:58:45.524+0000] {standard_task_runner.py:91} INFO - Job 267: Subtask extract_data
[2025-02-03T13:58:45.638+0000] {task_command.py:426} INFO - Running <TaskInstance: INETUM_ETL.extract_data manual__2025-02-03T13:58:38.091646+00:00 [running]> on host 2892ee60bb8a
[2025-02-03T13:58:46.115+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Andres Urrea' AIRFLOW_CTX_DAG_ID='INETUM_ETL' AIRFLOW_CTX_TASK_ID='extract_data' AIRFLOW_CTX_EXECUTION_DATE='2025-02-03T13:58:38.091646+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-03T13:58:38.091646+00:00'
[2025-02-03T13:58:46.118+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-02-03T13:58:51.470+0000] {dagto_csv.py:47} INFO - Proceso de extraccion de data realizado con exito
[2025-02-03T13:58:51.474+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-02-03T13:58:51.476+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-02-03T13:58:51.492+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=INETUM_ETL, task_id=extract_data, execution_date=20250203T135838, start_date=20250203T135845, end_date=20250203T135851
[2025-02-03T13:58:51.612+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-02-03T13:58:51.670+0000] {taskinstance.py:3482} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-02-03T13:58:51.705+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
